hasjob/views/index.py

```diff
@@ -338,13 +338,17 @@ def index(basequery=None, type=None, category=None, md5sum=None, domain=None,
     elif pinsandposts:
         g.impressions = {post.id: (pinflag, post.id, is_bgroup) for pinflag, post, is_bgroup in pinsandposts}
 
+    query_params = dict(request.args)
```

Comment:
Won't `dict` retrieve only the first value when multiple values are present? Like in `?l=bangalore&l=anywhere`?